scalaz/src/test/scala/org/atnos/eff/addon/scalaz/concurrent/TaskEffectSpec.scala

```diff
@@ -65,7 +65,7 @@ class TaskEffectSpec(implicit ee: ExecutionEnv) extends Specification with Scala
     } yield a + b
 
     eventually(retries = 5, sleep = 0.seconds) {
-      action[S].taskAttempt.runOption.runSequential must returnValue(beSome(beLeft(boomException)))
+      action[S].taskAttempt.runOption.runSequential must returnValue(Some(Left(boomException)))
```

Comment:
workaround ambiguous implicits. (Maybe specs2 issue? 🤔 )

```
[error]  found   : => Throwable
[error]  required: org.specs2.matcher.Expectable[?]
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method theValue in trait MustExpectations1 of type [T](t: => T)org.specs2.matcher.MustExpectable[T]
[error]  and method thisValue in trait ShouldExpectations of type [T](t: => T)org.specs2.matcher.ShouldExpectable[T]
[error]  are possible conversion functions from => Throwable to org.specs2.matcher.Expectable[?]
[error]       action[S].taskAttempt.runOption.runSequential must returnValue(beSome(beLeft(boomException)))
[error]                                                                                    ^
[error] one error found
[error] (Test / compileIncremental) Compilation failed
```

Reply:
This is annoying. Something has definitely changed in the implicit resolution. `beLeft` has 2 forms: one taking a parameters and another one directly returning a matcher. In 2.13 Scala seems to assume that we want the second form and tries to use the `.apply` method. I did a fix for this in specs2 but in the meantime in `eff` a workaround is to annotate a type and write `beSome(beLeft[Throwable](boomException))`.